aws_kube_params.inc

```diff
@@ -20,6 +20,7 @@ export NUM_WINDOWS_NODES=0
 export AWS_SSH_KEY=$HOME/.ssh/id_rsa
 
 export PRESET_INSTANCES_ENABLED=false
+export IS_PRESET_INSTANCES_DRY_RUN=false  # In Preset instance, if it's dry run, then it only generate script but not deploying. It has no impact to preset instances.
```

Comment:
your comments mentioned "It has no impact to preset instances"? please double check.

Reply:
Right, there is no impact.